### ISSUE TITLE
Webpack module imports using "~" are broken

### ISSUE THREAD

**Issue by raypatterson** [jtangelder/sass-loader#131]
Hi, 

I recently upgraded from 0.4.2 to 1.0.3 and [module imports](https://github.com/jtangelder/sass-loader#imports) stopped working. 

Downgrading back to 0.5.0 works but am unable to test 0.6.0 due to [this threading issue](https://github.com/sass/node-sass/issues/713).
(The `process.env.UV_THREADPOOL_SIZE = 100;` workaround doesn't work for 0.6.0 but does for 1.0.3).

The CSS output includes the correct resource paths within the imports but not the file contents.

`@import url(/absolute/path/to/file/in/node_modules/disk/filename.sass);`

My webpack config uses:

```
{ 
    ...
    "autoprefixer-loader": "^2.0.0",
    "css-loader": "^0.15.6",
    "extract-text-webpack-plugin": "^0.8.2",
    "sass-loader": "^0.5.0",
    "style-loader": "^0.12.3",
    "webpack": "^1.10.5",
    ...
}
```

Not sure if this is related to #108 or if it's an issue with node-sass itself. 

Any insight would be appreciated!

**Comment by jorrit** [comment]
Can you publish a small repository with this problem?

**Comment by jhnns** [comment]
Could you guys check out the current master branch and check if it works for you? I've heavily refactored the import mechanism and need more tests.

**Comment by jhnns** [comment]
Should be fixed with 2.0.0
